lsp-mode.el

```diff
@@ -7756,10 +7756,12 @@ This avoids overloading the server with many files when starting Emacs."
        (with-help-window (current-buffer)
          ,@(-map (-lambda ((msg form))
                    `(insert (format "%s: %s\n" ,msg
-                                    (if (with-current-buffer buf
-                                          ,form)
-                                        (propertize "OK" 'face 'success)
-                                      (propertize "ERROR" 'face 'error)))))
+                                    (let ((res (with-current-buffer buf
+                                                 ,form)))
+                                      (cond
+                                       ((eq res :optional) (propertize "OPTIONAL" 'face 'warning))
```

Comment:
`OPTIONAL` is a bit confusing, it doesn't indicate current validity of this feature. Perhaps this should be `NOT AVAILABLE` or `NOT AVAILABLE (OPTIONAL)`?

Reply:
done